File: 283-MoveZeroes/283-MoveZeroes.py
# Last updated: 6/22/2025, 11:04:15 AM
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        nz=0
        for i in range(len(nums)):
            if nums[i]!=0:
                nums[nz],nums[i]=nums[i],nums[nz]
                nz+=1
        
        

File: 283-MoveZeroes/283-MoveZeroes.java
// Last updated: 6/22/2025, 11:04:10 AM
class Solution {
    public void moveZeroes(int[] nums) {
        int temp=0;
        int i;
        for(i=0;i<nums.length;i++){
            if(nums[i]!=0)
            {
                nums[temp]=nums[i];
                temp++;
            }
        }
        for(i=temp;i<nums.length;i++)
        {
            nums[i]=0;
        }
  
    }
}